src/Utilities/MakeWithValue.hpp

```diff
@@ -17,7 +17,7 @@
 /// \ingroup DataStructuresGroup
 /// Implementations of make_with_value.
 namespace MakeWithValueImpls {
-template <typename R, typename T>
+template <typename R, typename T, typename ForSfinae = std::nullptr_t>
```

Comment:
I think in other places we just write `typename = std::nullptr_t`, omitting the name. I have no opinion on this though, just pointing it out.

Reply:
I agree, unless there is a compiler bug or some other reason it be named :)